### ISSUE TITLE
Release update?

### ISSUE THREAD

**Issue by magnetophon** [noseka1/linuxband#13]
I assume https://github.com/noseka1/linuxband/commit/c91304754ed421baa0f05df0339eb13f33e3ade7 is important.

Would you recommend creating a pkg from  the tarbal or from git?

**Comment by noseka1** [comment]
Yes, that commit is rather important and you want to have it included in your package. Would recommend to build the package directly from git.

**Comment by magnetophon** [comment]
Thanks.
I tried making a pkg from git, but it's missing install-sh.

**Comment by trebmuh** [comment]
Hi @noseka1 what about releasing a new version?
